ng2-components/ng2-activiti-analytics/index.ts

```diff
@@ -60,7 +60,11 @@ export const ANALYTICS_PROVIDERS: any[] = [
         DiagramsModule,
         MdTooltipModule,
```

Comment:
Please create a separate "material.module.ts" to store all angular/material references in a single place. That should help with unit tests as well. See https://github.com/Alfresco/alfresco-ng2-components/blob/development/ng2-components/ng2-alfresco-core/src/material.module.ts for an example.